modules/videoio/include/opencv2/videoio/legacy/constants_c.h

```diff
@@ -355,7 +355,8 @@ enum
     // Intel PerC streams
     CV_CAP_INTELPERC_DEPTH_GENERATOR = 1 << 29,
     CV_CAP_INTELPERC_IMAGE_GENERATOR = 1 << 28,
-    CV_CAP_INTELPERC_GENERATORS_MASK = CV_CAP_INTELPERC_DEPTH_GENERATOR + CV_CAP_INTELPERC_IMAGE_GENERATOR
+    CV_CAP_INTELPERC_IR_GENERATOR    = 1 << 27,
+    CV_CAP_INTELPERC_GENERATORS_MASK = CV_CAP_INTELPERC_DEPTH_GENERATOR + CV_CAP_INTELPERC_IMAGE_GENERATOR + CV_CAP_INTELPERC_IR_GENERATOR
```

Comment:
Please avoid any changes in "_c" files. It is here for compatibility with C++ code with legacy C constants only.

Reply:
Fix b41c0ea. Please tell me if squash is needed.